src/app/portlets/content-types/fields/content-type-fields-properties-form/content-type-fields-properties-form.component.html

```diff
@@ -0,0 +1,13 @@
+<form [formGroup]="form" (ngSubmit)="saveFieldProperties(form.value, form.valid)" novalidate>
```

Comment:
You don't need to pass 2 params, pass just ```form``` and inside the method get the props you need.